zipkin-autoconfigure/collector-kafka/src/main/java/zipkin2/autoconfigure/collector/kafka/ZipkinKafkaCollectorAutoConfiguration.java

```diff
@@ -11,7 +11,7 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package zipkin.autoconfigure.collector.kafka10;
+package zipkin2.autoconfigure.collector.kafka;
```

Comment:
note: all the autoconfigure types were package protected, meaning no-one could make a hard code dep on them without craziness